packages/signer-cli/README.md

```diff
@@ -0,0 +1,62 @@
+# @cennznet/signer-cli
+
+A simple CLI interface for offline signing. 
+
+## Create a transaction
+
+To create a transaction, you need to use the API to connect to a chain, enabling the creation of a transaction using the chain's metadata. In a terminal, run the `submit` command with the following form:
+
+`yarn run:signer submit --account <ss58> --ws <endpoint> <module.method> [param1] [...] [paramX]`
```

Comment:
can you please update these commands to show how it would be used by someone who installs the package from npm rather than someone who clones the source code
e.g. `npm i @cennznet/signer-cli` then `npx cennznet-signer-cli submit --account <ss58> --ws <endpoint> <module.method>` (not sure if that's correct)